src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -763,17 +763,15 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     //   normally. Namely, the keys we're concerned with are F7 down and Alt up.
     // Return value:
     // - Whether the key was handled.
-    bool TermControl::OnDirectKeyEvent(const uint32_t vkey, const bool down)
+    bool TermControl::OnDirectKeyEvent(const uint32_t vkey, const uint8_t scanCode, const bool down)

```

Comment:
BTW The `vkey` parameter could technically be an `uint8_t` as well.

Reply:
Totally. Didn't want to make too many changes in this PR for a hotfix :smile: